plugins/utility.py

```diff
@@ -248,6 +248,7 @@ def derpify(text):
 # colors
 @hook.command
 def color_parse(text):
+    """<string> -- Formats a string, replacing words wrapped in $( ) with actual colours or formatting."""
```

Comment:
The general format is `[args] - <doc>` (note the single hyphen rather than the double hyphen). (I know many plugins don't follow this format, but we're working on that).